README.md

```diff
@@ -192,6 +192,7 @@ using the matched rule and runs it. Rules enabled by default are as follows:
 * `tsuru_not_command` &ndash; fixes wrong tsuru commands like `tsuru shell`;
 * `tmux` &ndash; fixes `tmux` commands;
 * `whois` &ndash; fixes `whois` command.
+* `hdfs dfs` &ndash; Add the missing dash to the command.
```

Comment:
You actually have named your rule `hadoop_dfs_missing_dash` and that list was in alphabetic order.